src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/get-started.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: "Get started with Lambda containerized functions"
+metaDescription: Create Lambda containerized functions with New Relic.
+redirects:
+freshnessValidatedDate: never
+---
+
+import serverlessLambdaContainerizedFunctions from 'images/serverless_screenshot-full_lambda-containerized-functions.webp'
+
+Lambda containerized functions offer an alternative way to deploy code within AWS Lambda while still enjoying the benefits of serverless computing. 
+
+With New Relic, you can monitor AWS Lambda functions deployed using containerized images and: 
+
+* **Increase flexibility**: Use familiar container tooling and leverage pre-built images for your code and dependencies.
+* **Improve security**: Benefit from container isolation and potential security advantages associated with containerized environments.
+* **Ensure consistent runtime**: Ensure a constant development, testing, and deployment environment.
+
+<img
+    title="containerized functions"
+    alt="A screenshot for lambda containerized functions"
+    src={serverlessLambdaContainerizedFunctions}
+/>
+
+<figcaption>
+  To view your Lambda containerized functions,  Go to [<DoNotTranslate>**one.newrelic.com > All capabilities**</DoNotTranslate>](https://one.newrelic.com/all-capabilities) > Serverless functions. 
+</figcaption>
+
+To start using Lambda containerized functions, you can use Node.js 16.x or Node.js 18.x operating system. 
+
+<CollapserGroup>
+ <Collapser
+   id="node-16"
+   title="Create a Lambda containerized function with the Node.js 16. x agent"
+ >
+<Steps>
+<Step>
+### Install Python [#python]
```

Comment:
These are before starts requirements so better to add 217 & 218 lines in this requirements